### PR TITLE
Change default output style for multiple

### LINKED ISSUE #841: Duplicate status messages
**Describe the bug**
Same status messages are printed multiple times:
```
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-1 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-4 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-0 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-3 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-2 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-1 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-4 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-0 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-3 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-2 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-1 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-4 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-0 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-3 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-2 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-1 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-4 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-0 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-3 PENDING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-2 PENDING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-1 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-0 PENDING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m  0s matrix-12gn3h80wl2ll Pixel2-27 shard-2 PENDING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-1 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-1 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-1 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-1 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-1 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-1 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-1 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-1 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  1m 15s matrix-12gn3h80wl2ll Pixel2-27 shard-1 Preparing device.
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-3 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  1m 15s matrix-12gn3h80wl2ll Pixel2-27 shard-1 Preparing device.
  0m 23s matrix-12gn3h80wl2ll Pixel2-27 shard-4 RUNNING
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-0 RUNNING
  1m 21s matrix-12gn3h80wl2ll Pixel2-27 shard-3 Preparing device.
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  1m 15s matrix-12gn3h80wl2ll Pixel2-27 shard-1 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-4 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-0 Preparing device.
  1m 21s matrix-12gn3h80wl2ll Pixel2-27 shard-3 Preparing device.
  0m 29s matrix-12gn3h80wl2ll Pixel2-27 shard-2 RUNNING
  1m 15s matrix-12gn3h80wl2ll Pixel2-27 shard-1 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-4 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-0 Preparing device.
  1m 21s matrix-12gn3h80wl2ll Pixel2-27 shard-3 Preparing device.
  1m 32s matrix-12gn3h80wl2ll Pixel2-27 shard-2 Preparing device.
  1m 15s matrix-12gn3h80wl2ll Pixel2-27 shard-1 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-4 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-0 Preparing device.
  1m 21s matrix-12gn3h80wl2ll Pixel2-27 shard-3 Preparing device.
  1m 32s matrix-12gn3h80wl2ll Pixel2-27 shard-2 Preparing device.
  1m 15s matrix-12gn3h80wl2ll Pixel2-27 shard-1 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-4 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-0 Preparing device.
  1m 21s matrix-12gn3h80wl2ll Pixel2-27 shard-3 Preparing device.
  1m 32s matrix-12gn3h80wl2ll Pixel2-27 shard-2 Preparing device.
  1m 15s matrix-12gn3h80wl2ll Pixel2-27 shard-1 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-4 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-0 Preparing device.
  1m 21s matrix-12gn3h80wl2ll Pixel2-27 shard-3 Preparing device.
  1m 32s matrix-12gn3h80wl2ll Pixel2-27 shard-2 Preparing device.
  1m 55s matrix-12gn3h80wl2ll Pixel2-27 shard-1 Installing apps.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-4 Preparing device.
  1m 26s matrix-12gn3h80wl2ll Pixel2-27 shard-0 Preparing device.
  1m 55s matrix-12gn3h80wl2ll Pixel2-27 shard-3 Installing apps.
  1m 32s matrix-12gn3h80wl2ll Pixel2-27 shard-2 Preparing device.
  1m 55s matrix-12gn3h80wl2ll Pixel2-27 shard-1 Installing apps.
  2m  1s matrix-12gn3h80wl2ll Pixel2-27 shard-4 Installing apps.
  2m  1s matrix-12gn3h80wl2ll Pixel2-27 shard-0 Installing apps.
  1m 55s matrix-12gn3h80wl2ll Pixel2-27 shard-3 Installing apps.
  1m 32s matrix-12gn3h80wl2ll Pixel2-27 shard-2 Preparing device.
```

**To Reproduce**
Run Flank v20.06.0 using Fladle.

**Expected behavior**
Each status message should be unique.

**Details:**
```
version: v20.06.0
revision: 77f18db6109b49496fe82c99a553f16414f7366a
```
Fladle v0.10.1

### PR DESCRIPTION
**Pull request by jan-gogo**
Fixes #841 

Run flank with configuration that produces more than one test execution.

## Checklist

- [x] release_notes.md updated

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+- [#864](https://github.com/Flank/flank/pull/864) Change default output style for multiple. ([jan-gogo](https://github.com/jan-gogo))
 -
 -
 
 ## v20.06.2
 - [#853](https://github.com/Flank/flank/pull/853) Store @Ignore tests in the JUnit XML without sending ignored tests to FTL. ([piotradamczyk5](https://github.com/piotradamczyk5), [adamfilipow92](https://github.com/adamfilipow92))
-- [#853](https://github.com/Flank/flank/pull/858) Handle duplicated apk names. ([jan-gogo](https://github.com/jan-gogo))
--
--
+- [#858](https://github.com/Flank/flank/pull/858) Handle duplicated apk names. ([jan-gogo](https://github.com/jan-gogo))
```
Comment (bootstraponline) [inline release_notes.md:10]: ?

File: release_notes.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
 ## next (unreleased)
 - [#857](https://github.com/Flank/flank/pull/857) Added multimodule setup for test app. ([piotradamczyk5](https://github.com/piotradamczyk5))
 - [#837](https://github.com/Flank/flank/pull/837) Added obfuscate option to dump shards. ([piotradamczyk5](https://github.com/piotradamczyk5))
+- [#864](https://github.com/Flank/flank/pull/864) Change default output style for multiple. ([jan-gogo](https://github.com/jan-gogo))
```
Comment (bootstraponline) [inline release_notes.md:4]:
```suggestion
- [#864](https://github.com/Flank/flank/pull/864) Change the default output style to single when there are multiple matrices. ([jan-gogo](https://github.com/jan-gogo))
```

**Comment by bootstraponline** [comment]
based on the feedback in https://github.com/Flank/flank/issues/841, it sounds like this is a Fladle issue. We can keep the better user experience in Flank with default to multi.

File: release_notes.md (hunk 1 above)
Comment (jan-gogo) [reply release_notes.md:10]: Fix wrong pull request number from 853 -> 858

**Pull request closed without merging**